apps/aeprimop/src/aeprimop.erl

```diff
@@ -241,15 +241,14 @@ name_preclaim_tx_instructions(AccountPubkey, CommitmentHash, DeltaTTL,
     ].
 
 -spec name_claim_tx_instructions(pubkey(), binary(), non_neg_integer(),
-                                 fee(), nonce()) -> [op()].
-name_claim_tx_instructions(AccountPubkey, PlainName, NameSalt, Fee, Nonce) ->
+                                 non_neg_integer(), fee(), nonce()) -> [op()].
+name_claim_tx_instructions(AccountPubkey, PlainName, NameSalt, NameFee, Fee, Nonce) ->
     PreclaimDelta = aec_governance:name_claim_preclaim_delta(),
     DeltaTTL = aec_governance:name_claim_max_expiration(),
-    LockedFee = aec_governance:name_claim_locked_fee(),
     [ inc_account_nonce_op(AccountPubkey, Nonce)
     , spend_fee_op(AccountPubkey, Fee)
-    , lock_amount_op(AccountPubkey, LockedFee)
-    , name_claim_op(AccountPubkey, PlainName, NameSalt, DeltaTTL, PreclaimDelta)
+    , lock_amount_op(AccountPubkey, NameFee)
```

Comment:
This will be fixed in a later PR, it shouldn't be locked until the auction is over.